### DISCUSSION TITLE
New Bale Pickup option or unload way point

### DISCUSSION THREAD

**Discussion by Kiraxon** [Courseplay/courseplay#7247] [Q&A] [answered]
Hi,

I was wondering if it is possible to allow B train or multi trailer support for unloading bales at sell points. Your new bale pickup feature is awesome and from playing with it there is a Fliegel auto load transport tailer that will work with this feature, I tried a B train with dolly with two of these tailers but the dump way point seems to only trigger one trailer. 

I play on the Western Australian 4x map and it would be cool if I could use a B train and auto unload each trailer automatically through courseplay. Since you added bale pickup option I though I would put in my suggestion as maybe my suggestion could be make a reality. Australia uses B trains alot so hopefully you can consider my suggestion.

If it is already possible, and I might be doing something wrong please let me know. I did most of my learning from Youtubers. Here is the closest concept video I could find but I would like to try multi trailers: 

https://youtu.be/yaX3qd85ziw

I gave a time stamp spot in the video as there is other content not related to my question in the video.

Thanks

**Comment by Tensuko** [comment, marked as answer]
We will not support AutoLoad Trailers, as there is no standard scripts.

The Trailer that this guy in YT use is 'some kind of autoload'.
It basically use the same script as the normal bale collectors.
(pressing all they keys like working position, unload position, unload, etc)
Basically, it is GIANTS function.

https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=147304&title=fs2019
